docs/Data/Lists/FoldRight.md

```diff
@@ -0,0 +1,33 @@
+# `FoldRight<A, B>(fold, initialValue, list)`
+
+Folds a list from last to head.
```

Comment:
This workflow doesn't match the code. This matches `FoldLeft`.

Reply:
Fixed.